Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs

```diff
@@ -130,6 +131,22 @@ private void ExportUserFormModule(string path)
                                .ToArray();
             }
             File.WriteAllLines(path, code);
+
+            // LINQ hates this search, therefore, iterate the long way
+            foreach (string line in contents)
+            {
+                if (line.Contains("OleObjectBlob"))
+                {
+                    var resourceFileName = line.Trim().Split(new Char[] { '"' }).ElementAt(1);
+                    var tempName = Path.GetFileName(tempFile);
+                    var destPath = Directory.GetParent(path).FullName;
+                    if (File.Exists(Path.Combine(tempFilePath, tempFile)) && !destPath.Equals(tempFilePath))
+                    {
+                        File.Copy(Path.Combine(tempFilePath, tempName), Path.Combine(destPath, resourceFileName), overwrite:true);
```

Comment:
Shouldn't the file be *moved* instead? It's a temp file anyway, we don't want to leave useless files around... why copy it?